lib/Gocdb_Services/NotificationService.php

```diff
@@ -97,33 +93,14 @@ public function roleRequest($entity) {
         } else {
             // If the entity has valid users who can approve the role then send the email notification.
 
-            // Remove duplicate emails from array
-            $emails = array_unique ( $emails );
-
-            // Get the PortalURL to create an accurate link to the role approval view
-            $localInfoLocation = __DIR__ . "/../../config/local_info.xml";
-            $localInfoXML = simplexml_load_file ( $localInfoLocation );
-            $webPortalURL = $localInfoXML->local_info->web_portal_url;
-
-            // Email content
-            $headers = "From: no-reply@goc.egi.eu";
-            $subject = "GocDB: A Role request requires attention";
+            // Remove duplicate user ids from array
+            $authorising_user_ids = array_unique ( $authorising_user_ids );
```

Comment:
The following 'if' implies a null is allowed which would generate a useless php Warning here.